docs/working_with_masu.md

```diff
@@ -22,7 +22,8 @@ information.
 To create ROS-OCP data along with ROS -RHEL then we have to include
 below parameter:
 
-1.  `--ros-ocp-info        Generate ROS for Openshift data`
+1.  `--ros-ocp-info                   Generate ROS for Openshift data`
+1.  `--constant-values-ros-ocp        Generate constant values for ROS for Openshift data`
```

Comment:
I don't think we want to have it here, it looks like some example and typically we do not want these constant vlaues